tests/unit/executors/indexers/test_shelfindexer.py

```diff
@@ -31,3 +32,5 @@ def validate(req):
 
     with f:
         f.search([d], output_fn=validate)
+
+test_binarypb_in_flow()
```

Comment:
This invocation is not necessary if you run it via `pytest`. If you just want to test this test locally, you can also do `pytest tests/unit/executors/indexers/test_shelfindexer.py::test_binarypb_in_flow` which only invokes exactly this test.

Reply:
its a draft for easing testing locally